ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -65,41 +65,41 @@ interface AddressSuggestion {
 @Component({
     selector: 'ddp-address-embedded',
     template: `
-    <p *ngIf="block.titleText" class="ddp-address-embedded__title" [innerHTML]="block.titleText"></p>
-    <p *ngIf="block.subtitleText" class="ddp-address-embedded__subtitle" [innerHTML]="block.subtitleText"></p>
-    <ddp-address-input
-            (valueChanged)="inputComponentAddress$.next($event)"
-            [address]="inputAddress$ | async"
-            [addressErrors]="addressErrors$ | async"
-            [readonly]="isReadOnly$ | async"
-            [country]="staticCountry$ | async"
-            [phoneRequired]="block.requirePhone"
-            (componentBusy)="isInputComponentBusy$.next($event)"></ddp-address-input>
-    <ddp-validation-message
-            *ngIf="(errorMessagesToDisplay$ | async).length > 0"
-            [message]="(errorMessagesToDisplay$ | async).join(' ')">
-    </ddp-validation-message>
-    <form [formGroup]="suggestionForm" novalidate>
-        <mat-card id="suggestionMatCard" *ngIf="suggestionInfo$ | async as info">
-            <mat-card-header>
-                <mat-card-title translate>SDK.MailAddress.Suggestion.Title</mat-card-title>
-                <mat-card-subtitle translate>SDK.MailAddress.Suggestion.Subtitle</mat-card-subtitle>
-            </mat-card-header>
-            <mat-card-content>
-                <mat-radio-group class="suggestion-radio-group"
-                                 formControlName="suggestionRadioGroup">
-                    <mat-radio-button class="margin-5" value="suggested" [disableRipple]="true">
-                        <b>{{'SDK.MailAddress.Suggestion.Suggested' | translate}} </b>
-                        <span class="suggested"
-                              [innerHTML]="convertToFormattedString(generateTaggedAddress(info.entered, info.suggested,'b'))"></span>
-                    </mat-radio-button>
-                    <mat-radio-button class="margin-5" value="entered" [disableRipple]="true">
-                        <b>{{'SDK.MailAddress.Suggestion.AsEntered' | translate}} </b>{{ convertToFormattedString(info.entered) }}
-                    </mat-radio-button>
-                </mat-radio-group>
-            </mat-card-content>
-        </mat-card>
-    </form>`,
+        <p *ngIf="block.titleText" class="ddp-address-embedded__title" [innerHTML]="block.titleText"></p>
+        <p *ngIf="block.subtitleText" class="ddp-address-embedded__subtitle" [innerHTML]="block.subtitleText"></p>
+        <ddp-address-input
+                (valueChanged)="inputComponentAddress$.next($event)"
+                [address]="inputAddress$ | async"
+                [addressErrors]="verifyFieldErrors$ | async"
```

Comment:
A lot of whitespace changes, but this is the only real change in template. Renamed field for clarity.